index.html

```diff
@@ -6,8 +6,22 @@
     <title>Resque_a_pet</title>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/fonts.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/main.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/heading.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
 </head>
 <body>
-    
+ <div class="container">
+  <header>
+    <div class="header-logo"style="background-image: url('assets/images/logo.png');"></div>
+    <div class="header-photo"style="background-image: url('assets/images/the-dog-white-and-brown.jpeg');"></div>
```

Comment:
Nie piszemy styli inline.
Chyba że jest temu konkretny powód. W tym przypadku nie ma. Przenieś to do pliku .css

Reply:
Zrobione